projects/openthread/radio-receive-done-fuzz.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+max_len = 127
```

Comment:
I recommend to use 
  if (size > 127) return 0;
in the fuzz target instead of having this options file.